runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -688,53 +712,28 @@ private void checkClusterId(@Nonnull Layout layout) {
                     try {
                         IClientRouter router = getRouter(s);
                         // Try to get a layout.
-                        CompletableFuture<Layout> layoutFuture = router
-                                .getClient(LayoutClient.class).getLayout();
+                        CompletableFuture<Layout> layoutFuture =
+                                new LayoutSenderClient(router, Layout.INVALID_EPOCH).getLayout();
```

Comment:
why a new sender client here ?

Reply:
This is required on startup when we do not have an epoch to stamp the messages or a layout to fetch the client. So we need to create a new client here.